tenable/io/v3/vm/folders.py

```diff
@@ -0,0 +1,93 @@
+'''
+Folders
+=======
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`folders <folders>` API endpoints.
+
+Methods available on ``tio.v3.vm.folders``:
+
+.. rst-class:: hide-signature
+.. autoclass:: FoldersAPI
+    :members:
+'''
+from typing import List
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+
+
+class FoldersAPI(ExploreBaseEndpoint):
+
+    _path = 'api/v3/folders'
+    _conv_json = True
+
+    def create(self, name: str) -> int:
+        '''
+        Create a folder.
+
+        :devportal:`folders: create <folders-create>`
+
+        Args:
+            name (str):
+                The name of the new folder.
+
+        Returns:
+            :obj:`int`:
+                The new folder id.
```

Comment:
@tomeara The current response from the API doesn't have UUID. So, in this case, should we keep the type as ID?
`{
"unread_count":0
"custom":0
"default_tag":0
"type":"trash"
"name":"Trash"
"id":18
`}

Reply:
Yes, we can keep as an Int for now